README.md

```diff
@@ -250,6 +250,7 @@ Questions can be asked by raising an `Issue`.
 - [Raju Ghorai](https://github.com/coderj001)
 - [Ram Pandey](https://github.com/ram2510)
 - [Raman Singh](https://github.com/ramansingh189)
+- [Rahman Haroon](https://github.com/rahmanharoon)
```

Comment:
This is not in alphabetical order. Please place this line under `Raju Ghorai`'s line i.e. under the 250th line.